styles.css

```diff
@@ -1,20 +1,37 @@
 * {
   font-family: Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
 }
-
-h1 {
-  padding: 16px;
-  background-color: bisque;
-}
+/*
+nav {
+  position: fixed;
+  margin: 0px 10px 10px 0px;
+  padding: 0px 30px 200px 0px;
+}*/
 
 .navi {
   padding: 8px;
   background-color: b;
   border: solid;
   border-color: black;
   text-decoration: none;
+  width: 150px;
+
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  gap: 20 px;
```

Comment:
Hi Linda, I don't think your flexbox works as intended. Usually `display` flex is given on the parent, in your case the `nav` item. :)